src/perl/Permabit/BlockDevice/VDO.pm

```diff
@@ -600,20 +609,26 @@ sub installModule {
                                   modName         => $moduleName,
                                   modVersion      => $version,
                                   useDistribution => $self->{useDistribution},
+                                  useDnf          => $self->{useLinuxNextModule},
                                  );
   $module->load();
   $self->{_modules}{$machineName}{$moduleName} = $module;
   $self->_setModuleParameters();
 
-  $log->info("Installing module $VDO_USER_MODNAME $version on $machineName");
+  my $modVersion = $self->{useLinuxNextModule} ? $VDO_LINUXNEXT_MODVER
+                                               : $self->getModuleVersion();
+
   my $userModule
```

Comment:
In the upsteam case you're trying to implement, there is no module version number separate from the ernelversion. The argument is entirely meaningless in that case.

This means there is no reason to define VDO_LINUXNEXT_MODVER because the value should never be used for anything.